src/Functions/FunctionsConversion.h

```diff
@@ -1844,20 +1844,20 @@ class FunctionConvert : public IFunction
             mandatory_args.push_back({"scale", &isNativeInteger<IDataType>, &isColumnConst, "const Integer"});
         }
 
-        // toString(DateTime or DateTime64, [timezone: String])
+        // toString(DateTime or DateTime64, [timezone: String (const/non-const)])
```

Comment:
Could we remove "const/non-const" in l. 1847-1857? That's conceptually the default anyways. If needed, you can add a comment above l. 1860 that the timezone argument is deliberately non-const or const.